infra/cmake/modules/ExternalBuildTools.cmake

```diff
@@ -45,8 +45,6 @@ function(ExternalBuild_CMake)
   file(MAKE_DIRECTORY ${ARG_BUILD_DIR})
   file(MAKE_DIRECTORY ${ARG_INSTALL_DIR})
 
-  file(WRITE "${BUILD_STAMP_PATH}" "${PKG_IDENTIFIER}")
```

Comment:
It was designed to not retry build when failed. 

https://github.com/Samsung/ONE/blob/master/infra/cmake/modules/ExternalBuildTools.cmake#L27-L33

Please remove above comment.

Reply:
OK, it was by design...
I had a problem while make a cross build of external project and had to remove the stamp to make it retry the build.
So I think by design is like while development, the developer should manually clean the stamp.
Once the development is done, external build should not break in normal cases.
If it does break, then there is some critical situation and should skip retry as it will always fail...